### PR TITLE
return a constant value with reduce

### PR DESCRIPTION
**Pull request by gandhimena**

### CODE REVIEW

**Comment by soapyigu** [comment]
Agree with @iHackSubhodip , `acc` and `arg` look odd to me as well. Could you please use a more readable name @gandhimena ?

File: Array/SortArrayByParity.swift [hunk 1]
```diff
@@ -8,11 +8,9 @@
 
 class SortArrayByParity {
     func sortArrayByParity(_ A: [Int]) -> [Int] {
-        var outputArray = [Int]()
-        for (_,value) in A.enumerated(){
-            outputArray.insert(value, at: value % 2 == 0 ? 0 : outputArray.count)
+        return A.enumerated().reduce(into: [Int]()) { (acc, arg) in
```
Comment (soapyigu) [inline Array/SortArrayByParity.swift]: `acc` and `arg` look odd. Also, you can remove parentheses here.

Comment (wqfan) [inline Array/SortArrayByParity.swift]: I guess it's a question for the old change: why do we need to call `enumerated()` here?

Comment (iHackSubhodip) [reply Array/SortArrayByParity.swift]: Okay, There are ways. But It's all about, how we code 🗡

Comment (soapyigu) [reply Array/SortArrayByParity.swift]: You could rename the parameters to clarify themselves.

Comment (soapyigu) [reply Array/SortArrayByParity.swift]: I am ok to remove `enumerated()`.

**Comment by wqfan** [comment]
> * Leetcode compiler still doesn't support Swift 5.0 support. [isMultiple]

FYI, I opened a request two weeks ago and the Swift runtime is 5.0.1 now: https://support.leetcode.com/hc/en-us/articles/360011833974-What-are-the-environments-for-the-programming-languages

**Comment by iHackSubhodip** [comment]
> > * Leetcode compiler still doesn't support Swift 5.0 support. [isMultiple]
> 
> FYI, I opened a request two weeks ago and the Swift runtime is 5.0.1 now: https://support.leetcode.com/hc/en-us/articles/360011833974-What-are-the-environments-for-the-programming-languages

That's great news. @wqfan Thanks for your efforts.

**Pull request merged**